ExomeDepth/run_ExomeDepth.py

```diff
@@ -225,10 +241,11 @@ def call_cnv(args):
     os.makedirs(output_folder, exist_ok=True)
 
     """Determine gender"""
+    qc_status_gender = ''
```

Comment:
When are you using a single quote or double quote? If unsure, try to adapt previous choice. 
Same is true for other code changes, also in settings file.

Reply:
agree that it is is inconsistent. Changed fully in settings and run_ExomeDepth.py